Source/Session/Session.swift

```diff
@@ -35,6 +35,7 @@ public class Session: NSObject {
 
     private var currentVisit: Visit?
     private var topmostVisit: Visit?
+    private var disappearingVisitForSnapshotting: Visit?
```

Comment:
I've renamed `disappearingVisitForSnapshotting` to `previousVisit` in the feedback PR https://github.com/hotwired/turbo-ios/pull/189. 
cc @jayohms @joemasilotti What do you think about the name?